### DISCUSSION TITLE
\benchmark

### DISCUSSION THREAD

**Discussion by Matilde-rod** [joaomlourenco/novathesis#479] [Q&A] [answered]
Normalmente utilizo o Overleaf para qualquer LaTex related doc mas com a tese não consigo compilar em gratuito e tentei outras formas, incluindo o MikTeX, Authorea e TeXstudio.
Em todos o \benchmark dá-me erros e não avanço em nada.
Não encontro quase informação nenhuma sobre este código, para que serve em si, porquê os erros, faltam-me packages? Tenho de definir algo? 
Estou perdida, agradecia a quem me conseguisse ajudar :)

**Comment by joaomlourenco** [comment]
Que erro é que dá?
Comigo funciona sem problema com o MikTeK e o Tex-Live!

**Reply by Matilde-rod** [reply]
In TeXworks do MikTek:

! Undefined control sequence.
\benchmarktic ->\iow_term:e 
                            {\__benchmark_tictoc_prefix: TIC}\exp_args:NNf \...
l.120 \benchmarktic

And in TexStudio:
![g](https://github.com/joaomlourenco/novathesis/assets/157029810/2cbf0193-e189-4eb3-a259-36624fc1f596)

And in Authorea:
![a](https://github.com/joaomlourenco/novathesis/assets/157029810/7af8e36f-37e5-49d5-8348-5f9f44e69274)

**Reply by 1jgrodrigues** [reply]
Boas, a mim deu os mesmos erros, tive de desinstalar e voltar a instalar o miktek (no disco C:) e depois de atualizar no miktex console já deu. 
O miktex supostamente vai instalando packages à medida que precisa deles, talvez tbm possa estar a tentar correr no background e não estar a conseguir, acho que há uma opção ao instalar que instala os packages todos de uma vez, pode ser que ajude.

Espero que ajude

**Reply by 1jgrodrigues** [reply, marked as answer]
Na minha primeira instalação quando pedia para instalar os packages acho que lhe dei um diretório para o fazer, erro... Na segunda instalação deixei o instalar em sitio random (default) e já deu, pode ter sido o meu erro inicial.

**Reply by Matilde-rod** [reply]
Got it!! Obrigada :)
